lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -7,21 +7,18 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "Executor.h"
 #include "Memory.h"
+#include "MemoryManager.h"
 #include "SpecialFunctionHandler.h"
```

Comment:
This implements `SpecialFunctionHandler` so `SpecialFunctionHandler.h` should come first.